dt: 0.05
speed_scale: 2.0 # 速度缩放因子
Length: 1.3 # 船长
Width: 0.98 # 船宽
integrator_type: rk4
input_limit: 20.0
hydrodynamics:
  mass: 38.0
  Iz: 6.25
  B: 0.74
  Xu_dot: -1.9
  Yv_dot: -29.3171
  Nr_dot: -4.2155
  Xu: 26.43
  Yv: 72.64
  Nr: 22.96

wave:
  enable: true
  N: 64 # 采样点数
  Hs: 3.0 # 有义波高
  Tp: 6.5 # 波峰周期
  direction: 60.0 # 波向

wind:
  enable: true
  speed: 10.2 # 风速
  direction: 45.0 # 风向
  rho: 1.225 # 空气密度

topics:
  odom: /heron/odom
  target: /heron/goal
  target_odom: /heron/target_odom
  left_thruster_cmd: /heron/left_thruster_cmd
  right_thruster_cmd: /heron/right_thruster_cmd
